resources/views/index.blade.php

```diff
@@ -14,6 +14,7 @@
     <link rel="stylesheet" type="text/css" href="angular/account/register.css">
     <link rel="stylesheet" type="text/css" href="angular/about/about.css">
     <link rel="stylesheet" type="text/css" href="angular/admin/privacy.css">
+    <link rel="stylesheet" type="text/css" href="angular/home/welcome.css">
```

Comment:
Yeah, there's a lot of them.  I could put all the css in one file, but I don't particularly like that either.